roman.py

```diff
@@ -33,21 +33,21 @@ def get_roman_digit(number, base):
     if quotient >= 5:
         # do something including the half base
         if quotient == 9:
-            sys.stdout.write('%s%s' % (roman_prefix[(base*10)], roman_digit[(base*10)])) 
+            sys.stdout.write('%s%s' % (ROMAN_PREFIX[(base*10)], ROMAN_DIGIT[(base*10)])) 
         else:
-            sys.stdout.write('%s' % roman_digit[(base*5)])
+            sys.stdout.write('%s' % ROMAN_DIGIT[(base*5)])
             remaining = quotient - 5
-            sys.stdout.write('%s' % roman_digit[base] * remaining)
+            sys.stdout.write('%s' % ROMAN_DIGIT[base] * remaining)
     else:
         if quotient == 4:
             if base == 1000:
-                sys.stdout.write('%s' % roman_digit[base] * quotient)
+                sys.stdout.write('%s' % ROMAN_DIGIT[base] * quotient)
             else:
-                sys.stdout.write('%s%s' % (roman_prefix[(base*10)], roman_digit[(base*5)] )) 
+                sys.stdout.write('%s%s' % (ROMAN_PREFIX[(base*10)], ROMAN_DIGIT[(base*5)] )) 
         else:
-            sys.stdout.write('%s' % roman_digit[base] * quotient)
+            sys.stdout.write('%s' % ROMAN_DIGIT[base] * quotient)
 
-    get_roman_digit(remainder, base/10)    
+    get_ROMAN_DIGIT(remainder, base/10)    
 
 def get_largest_base(number):
```

Comment:
this is strange. not only is `base` not the correct term (i originally thought you were making it general enough to support conversion from binary, hex, etc), but this function can be done better with `log` and `floor` and stuff.

you are basically looking for the greatest power of ten less than the input